scripts/format_sources.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2016-2017 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+#
+
+#
+# This is a helper script for invoking uncrustify on a set of C++ files.
+# Invoking this script on a directory containing C++ files (*.cpp, *.h, *.c)
+# will format these files _in place_, according to an uncrustify config file.
+#
+# The $UNCRUSTIFY_CONFIG_FILE environment variable can be used to specify which
+# uncrustify config file to use.
+#
+
+COMPONENT_FOLDER="" # Set to softwarecontainer base dir
+UNCRUSTIFY_CONFIG_FILE="" # Add option to set this
```

Comment:
I had some idea that it would use the default config file unless specified, but that another config could be used with a flag/argument to "make uncrustify".